root.key

```diff
@@ -0,0 +1,5 @@
+-----BEGIN EC PRIVATE KEY-----
```

Comment:
why is this file needed?

Reply:
oh.. my bad. It is not needed. I thought it is not in my PR and was also wondering why git guardian is complaining about this file. I will remove it.